Added Cluster Queue/Jobs example to GraphQL Cookbook (#2224)
buildkite/docs: pages/apis/graphql/graphql_cookbook.md

}
```

## Clusters

A collection of common tasks with clusters using the GraphQL API.

> ### List cluster queues 

Reviewer: This should fix the linting error in the build:

```suggestion
### List cluster queues
```

> List the first 20 cluster queues' key and ID for the first 10 clusters in the organisation

Reviewer: I think this could be a bit simpler/clearer. Maybe:

```suggestion
For the first 10 clusters in an organization, list the key and ID of the first 20 cluster queues.
```

What do you think?

Author: In the latest commit 👍, TY @mbelton-buildkite

```graphql
query getClusterQueues {
  organization(slug: "organization-slug") {
    clusters(first:10){
      edges{
        node{
          name
          queues(first:20){
            edges{
              node{
                key
                id
              }
            }
          }
        }
      }
    }
  }
}
```

> ### List jobs as part of a specific cluster queue

Reviewer: Does this still keep the right meaning?

```suggestion
### List jobs in a cluster queue
```

Just looking if there's a sensible way to shorten it a bit.

Author: In the latest commit 👍, TY @mbelton-buildkite
